### PR TITLE
RHCOS bug batching 4.17

### PR DESCRIPTION
**Pull request by obrown1205**
Version(s):
4.17

Issue:
N/A

Link to docs preview:
https://82591--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-17-release-notes.html

QE review:
N/A

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-17-release-notes.adoc [hunk 1]
```diff
@@ -1584,6 +1584,12 @@ metadata:
 [id="ocp-4-17-rhcos-bug-fixes_{context}"]
 ==== {op-system-first}
 
+* Previously, LUKS encryption on a system using 512 emulation disks caused provisioning to fail at the `ignition-ostree-growfs` step because of an `sfdisk` alignment issue. With this release, the `ignition-ostree-growfs` script detects this situation and fixes the alignment automatically. As a result, the system no longer fails during provisioning. (link:https://issues.redhat.com/browse/OCPBUGS-35410[*OCPBUGS-35410*])
+
+* Previously, a bug in the `growpart` utility caused a LUKS device to lock. This prevented the system from booting and entering into an emergency mode. With this release, the call to the `growpart` utility is removed and the system successfully boots without issue. (link:https://issues.redhat.com/browse/OCPBUGS-33124[*OCPBUGS-33124*])
```
Comment (jeana-redhat) [inline release_notes/ocp-4-17-release-notes.adoc:1589]: Per the bug, I think this is more accurate:
```suggestion
* Previously, a bug in the `growpart` utility caused a LUKS device to lock. This caused the system to boot into an emergency mode. With this release, the call to the `growpart` utility is removed and the system successfully boots without issue. (link:https://issues.redhat.com/browse/OCPBUGS-33124[*OCPBUGS-33124*])
```

**Review by jeana-redhat — COMMENTED** [review]
Check this against the bug but I think it was sort of saying the opposite of eh issue reported as drafted. Otherwise LGTM

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

File: release_notes/ocp-4-17-release-notes.adoc (hunk 1 above)
Comment (obrown1205) [reply release_notes/ocp-4-17-release-notes.adoc:1589]: Good catch - ty!

**Pull request merged**